wpilibj/src/main/java/edu/wpi/first/wpilibj/Ultrasonic.java

```diff
@@ -79,7 +79,7 @@ public synchronized void run() {
   /**
    * Initialize the Ultrasonic Sensor. This is the common code that initializes the ultrasonic
    * sensor given that there are two digital I/O channels allocated. If the system was running in
-   * automatic mode (round robin) when the new sensor is added, it is stopped, the sensor is added,
+   * automatic mode (round-robin) when the new sensor is added, it is stopped, the sensor is added,
```

Comment:
Round Robin is the correct term

Reply:
im seeing both from multiple dictionaries, not sure on this one